src/table/components/TableBodyWrapper.jsx

```diff
@@ -8,6 +8,7 @@ import getCellRenderer from './renderer';
 import { getBodyStyle } from '../utils/styling-utils';
 import { bodyHandleKeyPress } from '../utils/handle-key-press';
 import { handleClickToFocusBody } from '../utils/handle-accessibility';
+import RootContext from '../../contexts/rootContext';
```

Comment:
this was the reason for creating the custom hook of `useRootContext` instead of importing two items on each use case